autowiring/Autowired.h

```diff
@@ -112,13 +110,10 @@ class Autowired:
   }
 
 private:
-  // The first deferred child known to need registration.  This is distinct from the m_pFlink entry in
-  // the base class, which refers to the _next sibling_; by contrast, this type refers to the _first child_
-  // which will be the first member registered via NotifyWhenAutowired.
-  std::atomic<AutowirableSlot<T>*> m_pFirstChild{nullptr};
-
   std::vector<autowiring::registration_t> m_events;
 
+  std::vector<autowiring::registration_t> m_autowired_notifications;
```

Comment:
I don't like this `std::vector` instance, though, and would normally try to find a way to remove it.  On the other hand, there is another identical vector right above it, so this is probably not too bad.